### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by sajjadrajaee**
Add quote and calculator pages,
Add links,
Using React Router.

### CODE REVIEW

File: src/components/navbar.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+export default function Navbar() {
+  return (
+    <div className="navbar">
+      <h1 className="navTitle">Magician Calculator!</h1>
+      <nav className="navLinks">
+        <Link className="navItem" to="/">Home</Link>
+        {' '}
+        |
+        {' '}
+        <Link className="navItem" to="/aboutCalculator">Calculator</Link>
```
Comment (danmainah) [inline src/components/navbar.js]: - [ ] @sajjadrajaee great work on your project, kindly ensure your link is correctly labeled, your calculator link is supposed to display the calculator but it's displaying your about calculator page.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @sajjadrajaee , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/navbar.js (hunk 1 above)
Comment (sajjadrajaee) [reply src/components/navbar.js]: Thank you @danmainah for your time and review of my code. I will fix it. 👍

**Review by whiz25 — APPROVED** [review]
Hi @sajjadrajaee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**